docs/_developer_onboarding.md

```diff
@@ -269,10 +269,42 @@ Here are the main points:
 
 #### Template and naming
 
-There is a template module to easily start a new feature. When creating a new module, you can just copy paste the template. It is
-located [here](../features/template).
-
-For the naming rules, please follow what is being currently used in the template module.
+This documentation provides you with the steps to install and use the AS plugin for generating modules in your project. 
+The plugin and templates will help you quickly create new features with a standardized structure.
+
+##### A. Installation
+
+Follow these steps to install and configure the plugin and templates:
+
+1. Install the AS plugin for generating modules :
+   [Generate Module from Template](https://plugins.jetbrains.com/plugin/13586-generate-module-from-template)
+2. Import file templates in AS :
+   - Navigate to File/Manage IDE Settings/Import Settings
+   - Pick the `tools/templates/file_templates.zip` files
+   - Click on OK
+3. Configure generate-module-from-template plugin : 
+   - Navigate to AS/Settings/Tools/Module Template Settings
+   - Click on + / Import From File
```

Comment:
The `+` is disabled :/

<img width="714" alt="image" src="https://user-images.githubusercontent.com/3940906/227982469-b6aa5ac3-2cef-4a8c-9552-7d395d3f87e5.png">